docs/automated-workflows/dagman-interaction.rst

```diff
@@ -117,6 +115,33 @@ The :tool:`condor_dagman` job places information about the status of the DAG
 into its own job ClassAd. The attributes are fully described in
 :doc:`/classad-attributes/job-classad-attributes`. The attributes are
 
+.. note::
+    Most of this information is also available in the ``dagman.out`` file.
+    Also, This information is updated every 2 minutes. Meaning information
```

Comment:
We can just remove the sentence "Meaning information  may be ...", as that's obvious.